hibernate/cache-commons/src/main/java/org/infinispan/hibernate/cache/commons/access/InvalidationSynchronization.java

```diff
@@ -32,7 +30,6 @@ public void beforeCompletion() {}
 
 	@Override
 	public void afterCompletion(int status) {
-      log.tracef("After completion callback with status %d", status);
```

Comment:
Why remove log message?

Reply:
I didn't find it too useful, and to keep it symmetric with LocalInvalidationSynchronization.

Reply:
I found it useful when debugging ISPN-8026. PFLValidator's `endInvalidatingKey` can be called from multiple places. This log message clarified for me what the origin of the end invalidation message was.